torchio/data/image.py

```diff
@@ -61,6 +61,11 @@ class Image(dict):
         tensor: If :py:attr:`path` is not given, :attr:`tensor` must be a 4D
             :py:class:`torch.Tensor` or NumPy array with dimensions
             :math:`(C, W, H, D)`.
+        h5DS: If both path and tensor aren't supplied, then an HDF5 Dataset has
```

Comment:
Can you please rename this variable to something more informative? Maybe `hdf5_dataset`.

Reply:
Sure